### ISSUE TITLE
Going back to search results from single results page is very slow

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#2182]
## Description

Going back to the search results from the single results page is very slow compared to visiting the single results page. Even on my very modern and fast laptop, the transition is noticeable. On my phone, the website feels broken, especially as different UI elements update at different times.

Here is a screen recording demonstrating the effect on my phone (a [Teracube 2e](https://myteracube.com/pages/teracube-2e#tech-spec-secs), definitely not the _slowest_ phone and still better than other low-spec phones.

https://github.com/WordPress/openverse/assets/24264157/3543702c-a9d8-4546-b78a-35b026adf536

And here on my partner's Samsung A21s, a much more common lower-spec phone. It seems to fair slightly better than the other phone but not by much.

https://github.com/WordPress/openverse/assets/24264157/8a93a2a0-739f-4df3-a3cd-b924424ced64

## Reproduction

1.  Navigate the Openverse website on a low-spec device. Go from a search results page to a single result page and then go back. Use either the back button on the browser or the "back to search results" button.
2.  See the different UI components updating at different times and the significantly long amount of time it takes to go back to the search results.

## Environment

- Device:  Listed above
- OS:  Android
- Browser:  Firefox nightly (on the teracube) and general release (on the samsung)

## Additional context

Possibly related to https://github.com/WordPress/openverse/issues/504

**Comment by zackkrida** [comment]
@obulat has a draft PR out which fixes many of the pop-in issues with the UI #2160, but I cannot yet tell if it addresses the core performance issue. There's definitely _something_ odd happening here. Even on a high-end android device with a gitabit connection I'm seeing a perceptible delay.

**Comment by obulat** [comment]
@sarayourfriend, can you still reproduce this issue? On my phone, the navigation is not instant, but it's not as bad as your screen recording.

**Comment by sarayourfriend** [comment]
@obulat I'm out of the house atm, but I have a TCL a507dl I can test this out with when I get home in a few hours.

**Comment by sarayourfriend** [comment]
I'm not seeing any discernible performance issues going between these pages on that phone, at least not ones that aren't characteristic of the phone overall (it struggles to do most anything :sweat_smile:).

Closing this as not reproducible.
